pkg/boot/multiboot/multiboot.go

```diff
@@ -258,13 +299,26 @@ func (m *multiboot) load(debug bool, ibft *ibft.IBFT) error {
 		m.mem.Segments.Insert(kexec.NewSegment(ibuf, r))
 	}
 
-	log.Printf("Preparing multiboot info")
-	if m.infoAddr, err = m.addInfo(); err != nil {
+	log.Printf("Preparing mu(l)tiboot info")
```

Comment:
thought here. Put a print in the if and in the else, and make it not have the ambiguity that comes with the (l). Would be nice to know exactly which it is at this point.